Sources/SwiftGRPC/Core/Metadata.swift

```diff
@@ -80,7 +80,16 @@ public class Metadata {
     defer { cgrpc_free_copied_string(valueData) }
     return String(cString: valueData, encoding: String.Encoding.utf8)
   }
-  
+
+  public func data(forKey key: String) -> Data? {
```

Comment:
nit: I think it would make more sense to move this into the extension at the bottom of the file, next to the subscript method.

Reply:
done